tox.ini

```diff
@@ -10,6 +10,7 @@ whitelist_externals=sh
 
 [testenv:py27]
 deps=
+        -rrequirements.txt
```

Comment:
Will rebase after https://github.com/release-engineering/cdn-definitions/pull/22 is merged